modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -3,5 +3,101 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-preparing-openshift-cluster_{context}"]
+= Preparing your cluster for {sandboxed-containers-first}
 
-= Preparing your cluster for OpenShift sandboxed containers
+Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
+
+* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers.
+// To Do: Add link to bare metal instructions: ../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
++
+[IMPORTANT]
+====
+* {sandboxed-containers-first} only supports {op-system} worker nodes. RHEL 7 or RHEL  8 nodes are not supported.
+* Nested virtualization is not supported.
+====
+* Your cluster must use installer-provisioned infrastructure.
+// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+
+[id="sandboxed-containers-additional-resource-requirements_{context}"]
+== Additional resource requirements for {sandboxed-containers-first}
+
+{sandboxed-containers-first} is a product that brings the ability to run workloads inside a sandboxed runtime, such as Kata Containers, to your {product-title} clusters. Each pod is represented by a virtual machine (VM). Each VM hosts a `kata-agent` process that acts as a supervisor for managing container workloads and processes running in those container. There are two additional processes that add more overhead:
```

Comment:
We really have three processes (and mention three later). What about adding `qemu-kiwi` as an additional process?

Reply:
"Each VM runs in a `qemu` process and hosts a `kata-agent` process...`